web_generator/bin/update_idl_bindings.dart

```diff
@@ -67,11 +67,13 @@ $_usage''');
 
   // Run app with `node`.
   final generateAll = argResult['generate-all'] as bool;
+  final inputFiles = argResult.rest;
```

Comment:
Hmm, instead of just treating the rest of the args as files, what if we used a multi-option?